test/acceptance/tests.py

```diff
@@ -389,64 +386,6 @@ def test_peer_assessment(self):
         self.do_peer_assessment()
 
 
-class PeerAssessmentTestStaffOverride(OpenAssessmentTest):
```

Comment:
Whoops, sorry about missing that comment. It seems like even though the tests are doing the same thing, they're also starting with different problem configurations. One of them is set up with `peer_only` and the other is set up with `self_only`. I would say it makes sense to keep them both in for coverage's sake.